samples/protected-setting-sample/README.md

```diff
@@ -0,0 +1,31 @@
+---
+page_type: sample
+languages:
+- yaml
+- json
+products:
+- azure-kubernetes-service
+- azure-marketplace
+---
+
+# Protected settings sample
+
+This folder contains samples that shows how ISV can create protected settings and input secret values in UIDefinition as well as consume them in ARM Template so that they can be passed to Kubernetes Extension Type as Configuration Protected Settings.
```

Comment:
It is good to add reference to the PasswordBox UI element so ISV can see all the properties they can change.